src/App.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
 import { Routes, Route } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
 import { useEffect } from 'react';
```

Comment:
- [ ] Good job, cards are displaying correctly on a mobile screen. However, the styling of the list on your _Homepage_ does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cards of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (or slightly alternating color shade) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/citec-47/React-capstone/assets/12745474/6ae3fc9c-7e33-4e9e-9fef-bad9273c27c1"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png">